pallets/identity/src/lib.rs

```diff
@@ -936,7 +955,10 @@ impl<T: Trait> Module<T> {
             );
             Self::link_key_to_did(&key, SignatoryType::External, identity_to_join);
         }
-
+        T::ProtocolFee::charge_fee(
+            &signer,
```

Comment:
I think the fee should be charged to the identity and not the signer in this case